Testscripts/Linux/perf_apache.sh

```diff
@@ -176,14 +177,14 @@ Run_ab()
 		concurrency_left=$(($concurrency_left - $concurrency_per_ab))
 		requests_left=$(($requests_left - $total_request_per_ab))
 		LogMsg "Running parallel ab command for: ${total_request_per_ab} X ${concurrency_per_ab}"
-		ab -n ${total_request_per_ab} -r -c ${concurrency_per_ab} http://${server}/test.dat >> ${log_folder}/${current_concurrency}.${testName}.bench.log & pid=$!
+		ab -n ${total_request_per_ab} -r -c ${concurrency_per_ab} -s $timeout http://${server}/test.dat >> ${log_folder}/${current_concurrency}.${testName}.bench.log & pid=$!
```

Comment:
what's the default timeout, if not specifed?

Reply:
Default is 30s.